architecture-building-systems/CityEnergyAnalyst
# 1978 create an uninstaller for the cea

This PR updates the CEA installer to also uninstall - a shortcut is added to the menu and the uninstaller tries to uninstall all the various CEA files (including the `cea.config` - the assumption being, if you're uninstalling, you're _done_ with the CEA!).

To test this:

- create a backup of your `cea.config` (it's in your profile folder)
- install from here: https://www.dropbox.com/s/mj7t2prgon2zotr/Setup_CityEnergyAnalyst_2.17a1.exe?dl=0
- uninstall
- check:
  - `cea.config` file should be gone
  - `Documents/CityEnergyAnalsyst` folder should be gone
  - Desktop shortcuts (if created) should be gone
  - start menu folder "City Energy Analyst" should be gone
